npm script issues - Installing from github vs. npm

# This is a Bug Report

### Updated Issue
I originally believed my problem was related to the directory structure and build commands of AWS Codebuild.  But I was also installing the package directly from your Github page like so:
`npm install -g https://github.com/serverless/serverless`

Upon switching back to:
`npm install -g serverless`

It installed!  (edit the npm package doesn't contain the scripts below) See below for some backgound thinking about the issue.  Now I am thinking that the npm scripts should work properly wether no matter the download location or OS, but am unsure if this is a problem with serverless or npm, so posting the issue here first for reference

Sls Version: 1.4
OS: AWS Codebuild Ubuntu - nodejs config 
Error Messages:
```
npm ERR! serverless@1.4.0 postinstall: `node /usr/local/lib/node_modules/serverless/scripts/postinstall.js`
npm ERR! Failed at the serverless@1.4.0 postinstall script 'node /usr/local/lib/node_modules/serverless/scripts/postinstall.js`
```

### Background of issue
We are using AWS codepipeline&codebuild CI/CD to deploy our serverless services.  A catch of using Codebuild is that every command is run from the root directory so if you need to run a command in a subdirectory you need to change directories and run the command in one line

Does not work:
```
cd subdirectory
node scripts/postinstall.js
```

Works:
```
cd subdirectory && node scripts/postinstall.js
```

My particular problem occurs from this [commit]( https://github.com/serverless/serverless/commit/daa223a2fc8f7a9afa538f576e59f8a49ca82d8f)

I believe it is because codebuild executes the npm script in the root directory of Codebuild, not the root directory of serverless.  If someone has some suggestions I would gladly attempt an implementation.

--- a/package.json
+++ b/package.json
@@ -34,6 +34,8 @@
   "files": [
     "bin",
     "lib",
+    "scripts/postinstall.js",
+    "scripts/preuninstall.js",
     "package.json",
     "npm-shrinkwrap.json",
     "README.md",

